plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/internal/AnalysesJobHandler.java

```diff
@@ -46,8 +54,14 @@ protected IStatus run(IProgressMonitor monitor) {
 			url = new URL("platform:/plugin/com.redhat.fabric8analytics.lsp.eclipse.ui/templates/index.html");
 			url = FileLocator.toFileURL(url);
 			IViewPart mainView = ExitHandler.getView();
-			jobId = ExitHandler.getJobId();
-			((StackAnalysesView) mainView).updatebrowserUrl(url.toString());
+			
+			if(!editorCheck) {
+				jobId = ExitHandler.getJobId();
+				((StackAnalysesView) mainView).updatebrowserUrl(url.toString());
+				
+			}
+			jobId = EditorComposite.jobID;
```

Comment:
What happens if you do have 2 editors open? Maybe you need 2 separate jobs for that.